.github/workflows/ci-test-go.yml

```diff
@@ -57,6 +57,21 @@ jobs:
       - name: Check out code into the Go module directory
         uses: actions/checkout@v3
 
+      - name: golangci-lint
+        # TODO: Remove each example/module once it passes the golangci-lint
+        if: ${{ inputs.platform == 'ubuntu-latest' && !contains(fromJSON('["examples/cockroachdb","examples/spanner","examples/toxiproxy","modules/compose","modules/pulsar","modules/redis"]'), inputs.project-directory) }}
```

Comment:
@mdelapenya , the condition for execution could be narrowed to enable less executions per module like on inputs.go-version or any other inputs.
What do you think ?